src/transforms/push.ts

```diff
@@ -148,7 +150,7 @@ export const processPush = async (github: GitHubInstallationClient, payload: Pus
 						authorTimestamp: githubCommitAuthor.date,
 						displayId: commitSha.substring(0, 6),
 						fileCount: files.length, // Send the total count for all files
-						files: filesToSend.map((file) => mapFile(file, repo, owner.name, sha.id)),
+						files: filesToSend.map((file) => mapFile(file, repo, owner.name || "", sha.id)),
```

Comment:
Since a **name** can be `null` do we want that to be the default?
I'm out of context a bit so non-issue if empty string it better.

Reply:
we should never use an empty string where undefined can be used.  Empty string just adds more complexity and a "third state" that we need to handle.

Also, was this a problem before?

Reply:
I have no idea "is there any problem at prod" right now. But from github's typing, that `name` could be `null` under the `owner`. 

And I have to do this because TS complaint about it (because I add a typing to `enqueuePush` method)

Reply:
Actually I think leaving it as it is is better, as the map file will produce `/some/path/null/some/file` instead of `/some/path//some/file`. Reverted.